docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -0,0 +1,982 @@
+
+With Storefront GraphQL's Faceted and Textual Search, you can:
+- Load category pages with no selections, including the both the facets and products relevant to the search results 
+- Load category pages with facet selections for specific facets
+- Load featured products from specific categories
+- Quicksearch products by using search terms (textual search)
+- Sort products alphabetically, by newest to oldest, etc
+
+You can access these features by querying the `SearchProducts` field. Note that you can query facets, filter by rating, or filter by "in-stock" only if the merchant is on a Pro or Enterprise plan. A merchant must [enable Product Filtering](https://support.bigcommerce.com/s/article/Product-Filtering-Settings?language=en_US#setup) for facets to be returned. In addition, only facets that a merchant marks as visible in their Product Filtering settings will be returned. 
+
+This page walks you through how to filter and query products and facets to display on your storefront. See [GraphQL Playground](/graphql-playground) for full schema documentation.  
+
+## How to Filter Products and Facets
+
+To use the Faceted and Textual Search feature, specify a filter in the argument for `SearchProducts`. For Faceted Search, you can filter by price, rating, among other features and attributes of products. For Textual Search, use the `searchTerm` field. 
+
+```GraphQL title=Filters for Products and Facets" lineNumbers
+...
+  searchProducts(
+  filters: {
+    searchTerm: "Sample"
+    price:{
+      minPrice:11,
+      maxPrice:200
+    },
+    rating:{
+      minRating:3,
+      maxRating:5
+    },
+    categoryEntityId:24,
+    searchSubCategories:false,
+    categoryEntityIds:[23],
+    brandEntityIds:[35],
+    productAttributes:[
+      {
+        attribute:"Color",
+        values:["Black"]
+      }
+    ],
+    isFreeShipping:true,
+    isFeatured:true,
+    isInStock:true
+  }
+  ... 
+) { 
+...
+```
+These filters affect **both** the products and facets that are returned. For example, filtering by rating returns only products that are within the specified rating range, as well as only facets that have products within the rating range. 
+
+See [GraphQL Playground](/graphql-playground) for descriptions of each filter. 
+
+## How to Get Products
+
+To get products, specify `products` as a field in `searchProducts`. Here is an example request that returns the first two products with a rating between three and five: 
+
+```GraphQL title="Request" lineNumbers
+query {
+  site {
+    search {
+      searchProducts(
+        filters: {
+          rating:{
+             minRating:3,
+             maxRating:5
+           }
+        }
+      ) {
+        products(first: 2) {
+          edges {
+            node {
+              entityId
+              name
+              prices {
+                price {
+                  value
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
```suggestion
```
&nbsp;
```

Reply:
disregard that backslash --

the comment is that these blocks will nest without a `&nbsp;` between them.

Reply:
Used new feature to nest while choosing the tab name